### PR TITLE
Revive fixed page

### PR DESCRIPTION
**Pull request by mtsmfm**

### CODE REVIEW

**Comment by okkez** [comment]
:+1:

File: app/models/fixed_page.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class FixedPage < ActiveRecord::Base
+  belongs_to :site
+end
```
Comment (tricknotes) [inline app/models/fixed_page.rb:5]: needs?

``` ruby
validates :slug, format: /\A\w+\z/, uniqueness: {scope: :site_id}
```

Comment (mtsmfm) [inline app/models/fixed_page.rb:5]: sir, yes, sir

**Comment by tricknotes** [comment]
nice :sparkles:

**Pull request merged**